packages/zipkin/index.d.ts

```diff
@@ -272,20 +268,8 @@ declare namespace zipkin {
     letContext<V>(ctx: TraceId, callback: () => V): V;
   }
 
-  namespace sampler {
-    class Sampler {
-      constructor(evaluator: (traceId: TraceId) => boolean)
-      shouldSample(traceId: TraceId): option.IOption<boolean>
-      toString(): String
-    }
-    function neverSample(traceId: TraceId): boolean
-    function alwaysSample(traceId: TraceId): boolean
-    class CountingSampler extends Sampler {
-      constructor(sampleRate: number)
-    }
-  }
-
-  class Request {
+  namespace Request {
```

Comment:
make a comment why namespace vs class?

Reply:
If there is a `constructor`, it should be a `class`.